DigitalLearningSolutions.Web/Controllers/RegisterController.cs

```diff
@@ -147,6 +168,7 @@ public IActionResult Summary()
             var centre = centresDataService.GetCentreName((int)data.RegisterViewModel.Centre!);
             var jobGroup = jobGroupsDataService.GetJobGroupName((int)data.LearnerInformationViewModel.JobGroup!);
             var viewModel = RegistrationMappingHelper.MapToSummary(data, centre!, jobGroup!);
+            AddCustomFieldsToViewBag(data.LearnerInformationViewModel, (int)data.RegisterViewModel.Centre!);

```

Comment:
Ah - do we not need to do this when posting as well?

Reply:
Whoops! You're absolutely right